### DISCUSSION TITLE
Probleme in Docker and Mac Bitwarden APP

### DISCUSSION THREAD

**Discussion by mesu0815** [dani-garcia/vaultwarden#3261] [General]
Hello, I have recently been having masive problems with my guitar.
In the log of the docker I have seen the following message that is written every second.
[parity_ws::io][INFO] Accepted a new tcp connection from 172.17.0.1:40094...
I use bitwarden with a PFSENSE and HAPROXY. 
The second problem is that I can no longer log on to the Mac with the programme.
Now I get the message after entering the master password:
Cannot read propertiers of null(reading "iterations").
On the IOS everything works as before, the WebClient also works without errors.
Can anyone give me a hint as to what could be going on?
 

Translated with www.DeepL.com/Translator (free version)

**Comment by BlackDex** [comment]
I suggest to install the latest version of Vaultwarden, you are using a very old version.

**Comment by mesu0815** [comment]
So do I musst to reinstall the Docker container? 
I make it on a Synology NAS. There stand under Version Lasted.
Or can I update the version in Docker?
I am not a Docker professional.
Thanks for help

**Reply by mesu0815** [reply]
updated to the latest version of the container successfull.  It was not that difficult.
Login via Mac works now, only the message in the LOG every second remains.

vaultwarden::api::notifications][INFO] Accepting WS connection from 172.17.0.1:47796

Someone has a tipp for this.
